docroot/profiles/custom/cgov_site/modules/custom/cgov_core/cgov_core.install

```diff
@@ -177,7 +177,7 @@ function cgov_core_themes_installed($theme_list) {
         $block_region = 'public_use';
         $block_label = 'Public Use';
         $block_html = <<<EOMarkup
-      <p><em>If you would like to reproduce some or all of this content, see <a href="/policies/copyright-reuse">Reuse of NCI Information</a> for guidance about copyright and permissions. In the case of permitted digital reproduction, please credit the National Cancer Institute as the source and link to the original NCI product using the original product's title; e.g., “[current-page:title] was originally published by the National Cancer Institute.”</em></p>
+      <p><em>If you would like to reproduce some or all of this content, see <a class="usa-link" href="/policies/copyright-reuse">Reuse of NCI Information</a> for guidance about copyright and permissions. In the case of permitted digital reproduction, please credit the National Cancer Institute as the source and link to the original NCI product using the original product's title; e.g., “[current-page:title] was originally published by the National Cancer Institute.”</em></p>
```

Comment:
This will need a manual content change when this goes live. @laurelthrash (or @andyvanavery31 ) can you start a document for the manual tasks. This would be kind of like what we had to do for some of the MVP items. (But the list should not be as long -- this might be the only thing)